app/lib/send_grid_client.rb

```diff
@@ -43,6 +43,9 @@ def send_email_with_template(options={})
             }
           ],
           "subject" => options[:subject],
+          "substitutions" => {
+            "-user_count-" => options[:user_count]
```

Comment:
Since this requires a string can you add a `.to_s` to this line to make sure we always pass a string to the API?